spec/controllers/accounts/registrations_controller_spec.rb

```diff
@@ -1,30 +1,36 @@
 require 'spec_helper'
 
 describe Accounts::RegistrationsController do
-  it "can get signup form" do
+  it 'can get signup form' do
```

Comment:
Is there a particular reason for changing to single quotes? standardrb seems to prefer double quotes:
> Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping